extensions/amp-story/1.0/page-advancement.js

```diff
@@ -463,7 +463,7 @@ export class ManualAdvancement extends AdvancementConfig {
 
         if (
           tagName.startsWith('amp-story-interactive-') &&
-          !this.isInScreenSideEdge_(event, this.element_.getLayoutBox())
+          !this.isInStoryPageSideEdge_(event, this.element_.getLayoutBox())
```

Comment:
Second arg, `this.element_.getLayoutBox()` can be removed here since you're using `storeService` in the function.

Reply:
Changed this bc offsetLeft & Top just weren't working on loading pages.